include/swift/Frontend/Frontend.h

```diff
@@ -433,6 +444,65 @@ class CompilerInstance {
   /// Frees up the ASTContext and SILModule objects that this instance is
   /// holding on.
   void freeContextAndSIL();
+
+private:
+  bool loadStdlibAndMaybeSwiftOnoneSupport();
+  ModuleDecl *importUnderlyingModule(ClangImporter *clangImporter);
+  ModuleDecl *importBridgingHeader(ClangImporter *clangImporter);
+
+  void fillInModulesToImportFromImplicitImportModuleNames(
+      SmallVectorImpl<ModuleDecl *> &importModules);
+  
+public: // for static functions in Frontend.cpp
+  
+  struct ImplicitImports {
+    SourceFile::ImplicitModuleImportKind kind;
+    ModuleDecl *objCModuleUnderlyingMixedFramework;
+    ModuleDecl *headerModule;
+    SmallVector<ModuleDecl *, 4> modules;
+    
+    ImplicitImports(CompilerInstance &compiler);
+  };
+  
+private:
+  void createREPLFileWithImports(ImplicitImports &implicitImports);
+  DelayedParsingCallbacks *computeDelayedParsingCallback();
+
+  void createMainFileWithImportsAndAddToModule(ImplicitImports &implicitImports);
+ 
+  void parseALibraryFile(
+      unsigned BufferID,
+      ImplicitImports &implicitImports,
+      PersistentParserState &PersistentState,
+      DelayedParsingCallbacks *DelayedParseCB);
+
+  /// Return true if had load error
+  bool parsePartialModulesAndLibraryFiles(
+      ImplicitImports &implicitImports,
+      PersistentParserState &PersistentState,
+      DelayedParsingCallbacks *DelayedParseCB);
+
+  void
+  checkTypesWhileParsingMain(PersistentParserState &PersistentState,
+                                     DelayedParsingCallbacks *DelayedParseCB);
+
+  OptionSet<TypeCheckingFlags> computeTypeCheckingOptions();
+
+  void parseAndTypeCheckTheMainFile(
+      PersistentParserState &PersistentState,
+      DelayedParsingCallbacks *DelayedParseCB,
+      const OptionSet<TypeCheckingFlags> TypeCheckOptions);
```

Comment:
No need for `const` on a value type.

Reply:
Happy to remove it. Thought I was being clearer by including it. Also removed two other places I had used it.